Cargo.toml

```diff
@@ -53,6 +53,7 @@ rust_decimal_macros = { version = "^1" }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 sea-orm = { path = ".", features = ["mock", "debug-print"] }
 pretty_assertions = { version = "^0.7" }
+time = { version = "^0.3", features = ["macros"] }
```

Comment:
Maybe merge with privies time?

Reply:
I don't want `macros` feature to be enabled by default. I just need it for `sea-orm` tests

Reply:
Ah, sorry, these are different files...

Reply:
NVM :D